package_manifest.yaml

```diff
@@ -40,11 +40,12 @@ copr_projects:
   vars:
     core_modules:
       - 'ruby:2.7'
-      - 'python39:3.9'
       - 'nodejs:14'
       - 'postgresql:12'
     core_buildroot_packages:
       - 'foreman-build'
+      - python3.11-rpm-macros
+      - python3.11
```

Comment:
As this change has to be done actively, can you split it out to it's own PR to go in first?